fpicker.c

```diff
@@ -140,6 +140,15 @@ bool check_config(fuzzer_config_t *config) {
             plog("[!] Fuzzer mode AFL not possible with USB device.\n");
             return false;
         }
+    } else if (config->device == DEVICE_REMOTE) {
```

Comment:
I used the same constraint I observed above for USB devices. Not sure this applies.